build.gradle

```diff
@@ -21,7 +21,7 @@ subprojects {
 }
 
 ext {
-    minSdkVersion = 9
+    minSdkVersion = 14
```

Comment:
Similarly, there doesn't seem to be a reason to change this

Reply:
This change is caused by upgrading robolectric to 4.3. It integrated the androidx.test framework, which requires min sdk 14.

Reply:
http://robolectric.org/migrating/